### DISCUSSION TITLE
Make it Possible to choose 800X600 from the GUI settings

### DISCUSSION THREAD

**Discussion by S4lXLV** [linuxdeepin/developer-center#3503] [Features Request & Ideas | 特性请求 & 头脑风暴]
Sometimes i change my resolution to 800x600 but there is no option for that in the GUI settings

**Comment by BLumia** [comment]
If I remember correctly, 800x600 resolution is not supported by DDE since most DDE applications cannot be properly displayed under this resolution 😂 

As a workaround, you can use `systemsettings` app from KDE or other tools to set the resolution of the monitor.

**Reply by S4lXLV** [reply]
ok Thank you
